### PR TITLE
Preminor bump

### PR DESCRIPTION
**Pull request by austin-rausch**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@jupiterone/jupiter-managed-integration-sdk": "^24.0.3",
+    "@jupiterone/jupiter-managed-integration-sdk": "24.3.0-secretsmanager.0",
```
Comment (aiwilliams) [inline package.json]: Is there a reason to avoid `25.0.0-secretsmanager.0`?

Comment (austin-rausch) [reply package.json]: No... and I thought thats what I bumped it to.

**Comment by austin-rausch** [comment]
Screwed up versions about 4 times but I got it now.

**Comment by austin-rausch** [comment]
Unpublished both versions I accidentally published.

**Pull request merged**